test/e2e/apimachinery/webhook.go

```diff
@@ -125,7 +125,12 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		cleanWebhookTest(client, namespaceName)
 	})
 
-	ginkgo.It("Should be able to deny pod and configmap creation", func() {
+	/*
+		Release : v1.16
+		Testname: LimitRange: CRUD
```

Comment:
copy paste

Reply:
There is also a very long todo at the end of this also long test... 

// TODO: add more e2e tests for mutating webhooks
	// 1. mutating webhook that mutates pod		// 1. mutating webhook that mutates pod
	// 2. mutating webhook that sends empty patch		// 2. mutating webhook that sends empty patch
	//   2.1 and sets status.allowed=true		//   2.1 and sets status.allowed=true
	//   2.2 and sets status.allowed=false		//   2.2 and sets status.allowed=false
	// 3. mutating webhook that sends patch, but also sets status.allowed=false		// 3. mutating webhook that sends patch, but also sets status.allowed=false
	// 4. mutating webhook that fail-open v.s. fail-closed		// 4. mutating webhook that fail-open v.s. fail-closed

Reply:
Those sound more like integration-level tests, exercising all the permutations of responses. I wouldn't really expect this test to expand to do all of those.